targets/UnrealMarketplacePlugin/examplesource/ExampleProject/Source/ExampleProject/PlayFabApiTests.h

```diff
@@ -318,15 +319,20 @@ class PlayFabApiTestSuite : public FAutomationTestBase
     FString CLOUD_FUNCTION_THROW_ERROR = TEXT("throwError");
 
     // Input from TestTitleData.json
-    TestTitleData testTitleData;
+    TestTitleData testTitleDataStorage;
```

Comment:
>testTitleDataStorage [](start = 18, length = 20)

I would keep the old name (just to change less code) but i don't insist, up to you :)

Reply:
I had a choice between this and changing the name of the local variable assigned from `GetTestTitleData()` in `LoadTestTitleData()`, in order to avoid a name-is-hidden warning.  I thought that changing the member to xStorage would be more clear than anything I could come up with for alternate names for the local.

Reply:
Change the local variable in the other location, and just use tempTestTitleData or localTestTitleData or myTestTitleData.
I consider it a "priority" thing, where this variable is more important than a local variable.

Reply:
Honestly though... they should never be different, never change for the life of this test, and it should only be loaded once...
So the local variable just shouldn't exist it all. Everything should be referencing this one.